### ISSUE TITLE
Example script `compare_tempo2_B1855.py` is broken

### ISSUE THREAD

**Issue by abhisrkckl** [nanograv/PINT#1353]
This script has the following issues:
1. It requires tempo2_utils. If this is not installed, the script is supposed to throw an exception pointing the user to the tempo2_utils Github repo. This error message is not shown correctly.
2. The par file used in this script (B1855+09_NANOGrav_dfg+12_TAI_FB90.par) is not included in the PINT distribution. There is a par file with a similar name (B1855+09_NANOGrav_dfg+12_TAI.par), but I'm not sure if they are the same or not. While using the available par file, the tempo2 and PINT residuals show a difference at 0.1 microsecond level. Is this expected? (See attachment
![J1855-example-plot](https://user-images.githubusercontent.com/20718391/180520789-25dac3ee-897a-4583-a41d-37acf74f76ad.png)
)

**Comment by dlakaplan** [comment]
there are a number of tests that require tempo2 to be installed, or have other dependencies.  in some cases these are old, or in other cases they can be used to generate output that is compared within the standard test suite.  I'm not sure which is the case here.  @aarchiba has been trying to remove deprecated material more broadly.

**Comment by abhisrkckl** [comment]
I think it is important to have example scripts and notebooks that just work. Otherwise, it can appear unreliable or intimidating to new users. I'll be happy to clean up the example scripts. 

In the case of this script in particular, what is its purpose? Is it supposed to show the difference between PINT and tempo2?

**Comment by dlakaplan** [comment]
I agree in general.  The examples that are more broadly intended for public consumption are those that are also notebooks on read-the-docs (although some others just take too long).  I think these scripts predate me and could potentially be removed, but it would also be good to just note in the docs (maybe a separate README in the `examples/` directory) what each file is for, what additional dependencies might be needed, etc?

**Comment by abhisrkckl** [comment]
Here is a list of broken example scripts. (By broken I mean I can't run them from the command line without any changes.)

`build_model_from_scratch.py`  --- Doesn't work from the command line, but works with Jupyter and ipython
`compare_tempo2_B1855.py` --- Missing par file
`compare_tempo2_J0613.py` --- Missing par and tim files
`fit_NGC6440E_MCMC.py` --- Missing par file (PR submitted)
`MCMC_walkthrough.broken` --- It's in the name :)

I agree with @dlakaplan that these should either be fixed or removed. Otherwise, they just serve to confuse new users.

**Comment by abhisrkckl** [comment]
It may be helpful to add a test to see if any of the example scripts are broken by a commit.

**Comment by dlakaplan** [comment]
For the ones included in the RTD build, that is done (the docs/notebooks test).  But there are others that are just in there.

**Comment by abhisrkckl** [comment]
OK... It will be nice to test all the included examples.

**Comment by aarchiba** [comment]
I suggest conversion of all the example scripts to notebooks, so people can see what they are doing on readthedocs, and so that breaking them will result in CI failures. It's usually pretty easy to do the conversion - iust stick cell breaks in obvious places.
